code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -827,7 +827,7 @@
 	var/mob/living/carbon/exposed_carbon = exposed_mob
 	var/obj/item/organ/stomach/ethereal/stomach = exposed_carbon.getorganslot(ORGAN_SLOT_STOMACH)
 	if(istype(stomach))
-		stomach.adjust_charge(reac_volume * REM * 20)
+		stomach.adjust_charge(reac_volume * REM * 30)
```

Comment:
why is this * REM? if we changed the value of REM no one is going to think of a mob exposure being affected.

Is it like this for other reagents?

Reply:
what? its Reagents Effects Modifier. What else would it do?

Reply:
it's a calculation thats based off metabolization so it makes less sense when applied to a non-metabolizing action.

better way to put it is if you changed REAGENTS_METABOLISM (used by REM calc) you arent expecting an exposure effect also gets changed.